src/components/common/PasswordInput/index.tsx

```diff
@@ -0,0 +1,113 @@
+"use client";
+
+import { useState } from "react";
+
+// import { Input } from "~/components/ui/input";
```

Comment:
please remove unnecessary comments in your app

Reply:
Done